src/V3Hasher.cpp

```diff
@@ -465,10 +465,12 @@ class HasherVisitor final : public VNVisitorConst {
         });
     }
     void visit(AstCellInline* nodep) override {
-        m_hash += hashNodeAndIterate(nodep, HASH_DTYPE, HASH_CHILDREN, [this, nodep]() {
-            m_hash += nodep->name();
-            iterateConstNull(nodep->scopep());
-        });
+        m_hash += hashNodeAndIterate(nodep, HASH_DTYPE, HASH_CHILDREN,
+                                     [this, nodep]() { m_hash += nodep->name(); });
+    }
+    void visit(AstCellInlineScope* nodep) override {
+        m_hash += hashNodeAndIterate(nodep, HASH_DTYPE, HASH_CHILDREN,
+                                     [this, nodep]() { m_hash += nodep->name(); });
```

Comment:
Why is this needed?  There's no name in the CellInlineScope same() method, which is inconsistent so seems wrong.  Also that class's name just looks under the cell's so perhaps the cellp is what should be used?

Reply:
I assumed the scope data needed to be encoded somewhere, should I add that? 

And will this even get traversed, since the AstScope method has ```iterateConstNull(nodep->aboveScopep());```, so should I remove this visitor method?

Reply:
I think you can remove the visitor.  This hashing is used in functions/expressions.